xml/admin_docupdates.xml

```diff
@@ -398,7 +398,7 @@
        <para>
         Added information about trusted boot support to
         <xref
-        linkend="sec.grub2.yast2.config.advanced.tab3"/> (Fate
+        linkend="sec.grub2.yast2.config.advanced.boot-code"/> (Fate
```

Comment:
why use - here instead of _?
also, this one is really long. at least the "advanced" seems unnecessary.
"yast2" should really be just "yast" at this point.
"grub2" can also be shortened to "grub", i think.

(all points also true for the other ids changed here.)